### DISCUSSION TITLE
Why does Firefly III look so similar to Snipe-It (or vice versa)?

### DISCUSSION THREAD

**Discussion by trallnag** [firefly-iii/firefly-iii#5500] [Firefly III] [answered]
I'm currently looking for a simple self-hosted solution to track all my "expensive" stuff like hard drives and cables and came across Snipe-It. I checked screenshots and noticed that the dashboard and the whole look and feel is pretty similar to Firefly.

Are both tools using a similar (frontend) stack or is one based on the other?

https://snipeitapp.com/

![image](https://user-images.githubusercontent.com/24834206/148396810-ba451819-cfe9-4003-9efe-f3aa5752e1b6.png)

**Comment by JC5** [comment, marked as answer]
We both use the same open source dashboard template.

https://adminlte.io/themes/AdminLTE/index2.html

**Reply by trallnag** [reply]
cool, thanks
